addons/web/static/src/js/libs/jquery.js

```diff
@@ -171,7 +171,7 @@ $.fn.extend({
             // Search for a body child which is at least as tall as the body
             // and which has the ability to scroll if enough content in it. If
             // found, suppose this is the top scrolling element.
-            if (bodyHeight - el.scrollHeight > 1) {
+            if (bodyHeight - el.scrollHeight > 1.5) {
```

Comment:
Damn, yes I already added the rounding error difference with https://github.com/odoo/odoo/commit/71b6fc7341dfec5e8fc2352c3143b1e324a880f8 but I just did it wrong I guess. 1.5 should do the trick :+1: